README.md

```diff
@@ -13,3 +13,8 @@ Start the mdoc watcher with `./scripts/server`. Once the documentation has compi
 If you want to view live changes, ensure you have the mdoc watcher running, then also start the docusaurus live reload with `docker-compose up`. Changes will be visible at http://localhost:3000.
 
 If you want to build an entirely static site for export to external hosting, run `./scripts/cibuild`.
+
+The usual development process can be the following:
```

Comment:
Looks like this was confusing and I don't think the development process is well described. I'd like to avoid giving the same instructions in multiple places (so that if we have to edit them later we only need to edit one spot). 

What if we replaced the entire contents of the README with something like this (feel free to tweak as desired):
```
# GeoTrellis Documentation

This repo is the home of the GeoTrellis documentation. It does not yet have a home on the web.

`./docs` contains the editable markdown documentation which is then built with [mdoc](https://scalameta.org/mdoc/).

`./website` contains the static site that renders the markdown documentation and is built with [Docusaurus](https://docusaurus.io/).

## Developing

In order to edit and build the markdown docs, you'll need to start the mdoc watcher with `./scripts/server`. 

If you also want to view the rendered output on the static site, you'll need to start the docusaurus local dev server with `docker-compose up` after you've started the mdoc watcher. The static site will be visible at http://localhost:3000.
```

Reply:
I'll just remove this extra line that was helpful for me.